charts/vault-monitoring/templates/servicemonitor.yaml

```
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ include "vaultMonitoring.name" . }}
+  labels:
+    {{- include "vaultMonitoring.labels" . | nindent 4 }}
+{{- if .Values.vault.serviceMonitor.labels }}
+    {{- toYaml .Values.vault.serviceMonitor.labels | nindent 4}}
```

Didn't you rename lot's of stuff from `vault` to `vaultMonitoring` at some point?
```suggestion
    {{- toYaml .Values.vaultMonitoring.serviceMonitor.labels | nindent 4}}
```